### ISSUE TITLE
RFP-David Jacobson (BMO Harris)

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/100k-Mentor-Challenge-Outreach#218]
**User Story / Problem / Details**
The Content Editor is in charge of ensuring the initial relationship we foster with an interviewee does not decay over time for interviewees that we designate as important. As a result we want to follow a process that allows us to interact and build a relationship with the interviewee over time. The final step of the process should be focused on how connect with an interviewee over multiple interactions and over the course of a year.

**Technical Definition of Done**
Complete all steps in the process

**Definition of Ready**
All of the steps in the process have been approved by Brett 

**Process**
- [ ] Review the interview in its entirety 
- [ ] Thank you note to the participant
- [ ] Quick clip (30s) that shows the interviewee some of the raw footage/a particularly interesting part of the discussion
- [ ] ID potential contacts
- [ ] Formulate ask
- [ ] Create a Relationship Action Plan

**Comment by howie-emond** [comment]
Clip
(its the chicken soup one, same as short clip for tiktok)

A: 38:00-38:26
Q: 41:03-41:20
A: 41:20-41:29
A: 42:12-42:18
Total: 58s

**Comment by howie-emond** [comment]
Hi David, we wanted to reach out and thank you again for participating in our Executive Interview Series. 

During our discussion, we had the privilege of hearing about your storied career, making the most of your network, imposter syndrome, and so much more. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The points you made on building an engaging, intriguing, and noticeable resumes will help our students be more effective in displaying their qualities.

At the end of our discussion, we had then opportunity to introduce you to our mentorship program. We've identified a few targets from your network that we'd be grateful to get involved. Again, we thank you for your time and meaningful conversation, and look forward to building a closer relationship with you and your colleagues.
